paddle/fluid/operators/mkldnn/layer_norm_mkldnn_op.cc

```diff
@@ -133,20 +132,7 @@ class LayerNormMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
 
     std::shared_ptr<mkldnn::memory> scaleshift_memory;
     if (with_scaleshift) {
-      auto scale_tz = paddle::framework::vectorize(scale->dims());
-      const unsigned int C = scale_tz[0];
-
-      // MKLDNN requires a single piece of memory for scale and shift/bias
-      // data
-      std::vector<float> scaleshift_data;
-      scaleshift_data.reserve(2 * C);
-      scaleshift_data.insert(scaleshift_data.begin(), scale->data<float>(),
-                             scale->data<float>() + C);
-
-      scaleshift_data.insert(scaleshift_data.end(), bias->data<float>(),
-                             bias->data<float>() + C);
-
-      scaleshift_memory = handler.AcquireScaleShiftMemory(scaleshift_data);
```

Comment:
I saw 
```
// TODO(jczaja): Enable FP32 when performance is good
namespace ops = paddle::operators;
REGISTER_OP_KERNEL(layer_norm, MKLDNN, ::paddle::platform::CPUPlace,
                   ops::LayerNormMKLDNNOpKernel<paddle::platform::bfloat16>);
```
in the last few lines of this file. So does this PR improve the performance and hence could register FP32 kernel layer_norm ?

Reply:
Unfortunately not. I have deleted one unnecessary copying of scaleshift tensor, but that have improved the performance by around ~1%, so this PR is more like refactoring. Our layernorm is still almost 2x slower than native kernel and that will need further investigation